swagger/v1/framework_response_reference.yaml

```diff
@@ -10,6 +10,9 @@ FrameworkResponseReference:
       - type: 'null'
       - type: array
         description:  Multiple objects returned if resource is associated to multiple framework_responses
+      example:
+        - id: ea5ace8e-e9ad-4ca3-9977-9bf69e3b6154
+          type: framework_responses
       required:
```

Comment:
Is indent right?

Reply:
Unfortunately if I move it into array it won't pull this example out when rendering the docs, and will either pull the single example or default to it's own (with removing the specific examples specified below)